src/browser/modules/App/keyboardShortcuts.ts

```diff
@@ -0,0 +1,94 @@
+import { useEffect } from 'react'
+import { FOCUS, EXPAND, CARDSIZE } from 'shared/modules/editor/editorDuck'
+import { Bus } from 'suber'
+
+export const isMac = /Mac|iPad/.test(navigator.platform)
+const modKey = isMac ? 'metaKey' : 'ctrlKey'
+
+type ModifyerKey = 'metaKey' | 'altKey' | 'ctrlKey'
+const printModifyers: Record<ModifyerKey, string> = {
+  altKey: isMac ? '⌥' : 'alt',
+  metaKey: isMac ? '⌘' : 'win',
+  ctrlKey: isMac ? '⌃' : 'ctrl'
+}
+
+const encodeKey: Record<string, number> = { c: 67, f: 70, '/': 55 }
```

Comment:
I don't understand these values, they are the ascii values for C, F and 7? I think there's a mistake, I can focus the editor by pressing `7` now, but not the `/` key on this branch.

Reply:
They are the event keycode values, which I also thought were ascii values but turns out they aren't. Pressing the cmd+alt+c here shows keycode 67 instead of 99 which from "c".charCodeAt(0)..

Reply:
I think they are ascii values, but for uppercase C and F